app/assets/javascripts/editor/form.js.es6

```diff
@@ -1,7 +1,9 @@
-window.setupPostForm = () => {
-  function applySelect2($element, options) {
+$(() => {
+  function applySelect2(selector, options) {
```

Comment:
It seems `options` is currently unnecessary.
When we want to pass some options, `$element.data()` is available.